pkg/handler/handler.go

```diff
@@ -692,13 +694,38 @@ func (h handler) lastoperation(w http.ResponseWriter, r *http.Request, params ma
 		return
 	}
 
+	// we have a binding job
+	if strings.Index(r.URL.Path, "/service_bindings/") > 0 {
```

Comment:
`strings.Contains` [might](https://golang.org/pkg/strings/#Contains) be more helpful when reading this? it is basically just a wrapper around what you are doing though so meh.

Reply:
Using ```strings.Contains``` would make the code read a little better.